frontend/src/components/Form.js

```diff
@@ -91,6 +91,14 @@ export default function Form(props) {
   const [savePreference, setSavePreference] = React.useState(true);
   const [matchPreference, setMatchPreference] = React.useState('none');
 
+  /**
+   * @return {Boolean} true/false based on completion status of personal fields
+   */
+  function checkPersonalPreferenceFields() {
+    // TO-DO(#65): Add interests to this validation
+    return role === '' || productArea === '';
+  }
```

Comment:
So if both role and product area are empty then the fields are complete?

Reply:
Yes. So you no longer have to submit a role and product area (and later interests as well) to submit the match form. In the case that you do not submit role and PA, the "similar" and "different" match preference radio buttons are disabled:
![Screenshot 2020-07-22 at 11 17 09 AM](https://user-images.githubusercontent.com/32528244/88194550-f3ca0b80-cc0c-11ea-9307-5206ed70944a.png)

As soon as a role and PA are selected, you are able to submit a preference again!

Reply:
Also, is the UI change we discussed earlier about dividing the "choose your match preferences" section into two (Describe yoruself, Who do you want to meet?) coming in a later PR?

Reply:
I guess it's not clear to me when this method should return true. The name suggests it should return true if both fields are filled out but instead it returns true if both fields are empty.

Reply:
> Also, is the UI change we discussed earlier about dividing the "choose your match preferences" section into two (Describe yoruself, Who do you want to meet?) coming in a later PR?

Yes, this will come in a later PR